packages/split-layout/src/vaadin-split-layout.js

```diff
@@ -248,7 +246,11 @@ class SplitLayout extends ElementMixin(ThemableMixin(mixinBehaviors([IronResizab
 
   /** @private */
   _processChildren() {
-    this.getEffectiveChildren().forEach((child, i) => {
+    const effectiveChildren = FlattenedNodesObserver.getFlattenedNodes(this).filter(
+      (node) => node.nodeType === Node.ELEMENT_NODE
+    );
```

Comment:
Out of curiosity, is there any particular reason we need this and not `this.children`?

Reply:
I guess the original reason was to support child elements slotted from a wrapper component, something like:

```html
<wrapper-layout>
  #shadow-root
    <vaadin-split-layout>
      <slot></slot>
    </vaadin-split-layout>
    
  <div>First</div>
  <div>Second</div>
</wrapper-layout>
```

I started adding a test case for it and discovered that child elements can't actually be slotted like this even on master, so effectively the feature doesn't work the way it was probably originally intended to. Changed to use `this.children` instead, as suggested.

Reply:
Now I see, thanks. Yes that's probably not a supported use case indeed.